app/scripts/lib/SnapsNameProvider.ts

```diff
@@ -135,9 +135,12 @@ export class SnapsNameProvider implements NameProvider {
         },
       )) as AddressLookupResult;
 
-      const domain = result?.resolvedDomain;
+      const domains = result?.resolvedDomains;
 
-      proposedNames = domain ? [domain] : [];
+      // TODO: Determine if this is what we want.
```

Comment:
What do we need to determine here?

Reply:
We need to determine how we are gonna map from the new return format to the format that the Petnames system expects.

@hmalik88 is gonna be working on that after this PR is merged, just left it as a note: https://github.com/MetaMask/MetaMask-planning/issues/1959